### PR TITLE
Add create-cluster-management to Makefile

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:
This PR adds a new Makefile command to easily spin up a new Management cluster and the test1 example cluster.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -238,6 +238,35 @@ create-cluster-ha: binaries-dev ## Create a development Kubernetes cluster on AW
 	-p ./cmd/clusterctl/examples/aws/out/provider-components.yaml \
 	-a ./cmd/clusterctl/examples/aws/out/addons.yaml
 
+.PHONY: create-cluster-ha-management
+create-cluster-ha-management: ## Create a development Kubernetes cluster on AWS using HA examples in a KIND management cluster.
```
Comment (detiber) [inline Makefile]: afaict this only creates a single control plane machine and not HA as this example says.

Comment (vincepri) [reply Makefile]: True, need to fix it :D

Comment (vincepri) [reply Makefile]: Fixed!

**Pull request merged**